src/components/Card.js

```diff
@@ -15,13 +27,32 @@ export default class Card {
   }
 
   _handleLike = () => {
-    this._likeButton.classList.toggle("card__like-button_active");
+    if (this.isLiked) {
+      this._unlikeHandler(this);
+      this.isLiked = false;
+    } else {
+      this._likeHandler(this);
+      this.isLiked = true;
+    }
   };
 
+  addLike() {
+    this._likeButton.classList.add("card__like-button_active");
+  }
+
+  removeLike() {
+    this._likeButton.classList.remove("card__like-button_active");
+  }
+
   _handleDelete = () => {
+    this._deleteHandler(this);
+  };
+
+  removeCard = () => {
     this._element.remove();
     this._element = null;
   };
+
   _setEventListeners() {
     this._element
       .querySelector(".card__image")
```

Comment:
`      .querySelector(".card__image")
`

You've found the image in line 71 as `this._cardPicture` and it's available here

Reply:
* You haven't corrected something https://github.com/ssbrandt/se_project_aroundtheus/pull/1#discussion_r1323382799  




```js

    this._cardPicture.addEventListener("click", this._handleCardClick);

```

Reply:
Fixed.